components/SearchBox/index.js

```diff
@@ -19,14 +17,14 @@ var SearchBox = React.createClass({
 
     return (
       <input type="text"
-      placeholder={this.props.placeholder}
-      name="algolia-query"
-      className={classNames}
-      data-role="autocomplete"
-      autoComplete="off"
-      autoFocus="autofocus"
-      onChange={this.change}
-      role="textbox" />
+        placeholder={this.props.placeholder}
+        name="algolia-query"
+        className={classNames}
+        data-role="autocomplete"
```

Comment:
I think this data-role is probably useless. I guess there should be a `role="textbox"` and we should use `aria-autocomplete="list"`